server/routerlicious/packages/lambdas/src/scriptorium/lambda.ts

```diff
@@ -134,17 +167,16 @@ export class ScriptoriumLambda implements IPartitionLambda {
 				}
 
 				metric?.setProperty("status", status);
-				metric?.success(
-					`Scriptorium completed processing and checkpointing of batch with offset ${batchOffset?.offset}`,
-				);
+				metric?.success("Scriptorium completed processing and checkpointing of batch");
 
 				// continue with next batch
 				this.sendPending();
 			},
 			(error) => {
 				// catches error if any of the promises failed in Promise.all, i.e. any of the ops failed to write to db
 				status = ScriptoriumStatus.ProcessingFailed;
-				const errorMessage = `Scriptorium failed to process batch with offset ${batchOffset?.offset}, going to restart`;
+				metric?.setProperty("timestampProcessingFailed", Date.now());
```

Comment:
Date.now() would return the number of milliseconds since Epoch:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/now

Was that intended? If not, and you want the ISO UTC timestamp, you can do something like this:

```js
const date1 = new Date();

console.log(date1);
// Mon Mar 13 2023 12:33:04 GMT-0700 (Pacific Daylight Time)

console.log(date1.toISOString());
// "2023-03-13T19:33:04.892Z"

```

Reply:
Agree and updated the same :)